yaml/Granta.BomAnalyticsServices.V1.yaml

```diff
@@ -3421,6 +3421,8 @@ components:
     Common.Sustainability.MaterialContributingComponent:
       type: object
       properties:
+        ParentName:
```

Comment:
Am I missing the point of versioned APIs here? The V1 api has changed here, as well as the V2 api?